theme/accessibility.less

```diff
@@ -13,7 +13,8 @@
     outline: 1px solid transparent !important;
 }
 
-@media not all and (pointer:coarse) {
+@media not all and (pointer:coarse), /* If not touch screen */
+all and (-ms-high-contrast: none), (-ms-high-contrast: active) { /* Or if IE 10+ */
```

Comment:
Tested on Firefox? According to this, pointer:course is not supported on Firefox. I would also test Chrome on IOS, I believe that doesn't work on it either. 
http://caniuse.com/#search=pointer%3Acoarse

Reply:
I'm investigating a better solution without breaking IE (which is tricky).